ohif-configurable/hp-extension/src/custom-attribute/sameAs.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * This function extracts an attribute from the already matched display sets, and
+ * compares it to the attribute in the current display set, and indicates if they match.
+ * From 'this', it uses:
+ *    `sameAttribute` as the attribute name to look for
+ *    `sameDisplaySetId` as the display set id to look for
+ * From `options`, it looks for 
+ */
+export default function (displaySet, options) {
+  const { sameAttribute, sameDisplaySetId } = this;
+  if( !sameAttribute ) {
+    console.log("sameAttribute not defined in", this);
+    return `sameAttribute not defined in ${this.id}`;
```

Comment:
just for curiosity, is there any difference if I return a string or boolean?
It seems this message is being used only for logging purpose